tests/jobs/test_scheduler_job.py

```diff
@@ -3812,6 +3812,20 @@ def test_mapped_dag(self, dag_id, session):
         dr.refresh_from_db(session)
         assert dr.state == DagRunState.SUCCESS
 
+    @pytest.mark.long_running
+    @pytest.mark.xfail(reason="Taskflow mapped tasks don't take default_args yet")
+    @pytest.mark.parametrize(
+        "dag_id", ["test_mapped_slas_disabled_classic", "test_mapped_slas_disabled_taskflow"]
+    )
+    def test_mapped_dag_slas_disabled(self, dag_id, session):
+
+        dag_file = TEST_DAGS_FOLDER / f'{dag_id}.py'
+        self.dagbag.process_file(str(dag_file))
```

Comment:
This doesn't need to be in the scheduler job to test it -- we can make this a test that the exception is thrown in tests/models/test_baseopeator.py.

Specifically we don't to add new test files (which can bet loaded by other tests slowing them down) nor do we need to check it makes it in to the import_errors -- that behaviour is already tested elsewhere.

Reply:
ok sure yeah was not exactly sure where best to put. will do that.